elm-package.json

```diff
@@ -1,28 +1,26 @@
 {
-    "version": "0.2.0",
+    "version": "1.0.0",
     "summary": "feed the tree some ether.",
     "repository": "https://github.com/cmditch/elm-web3.git",
     "license": "BSD3",
     "source-directories": [
-        "src"
-    ],
-    "exposed-modules": [
-        "Web3",
-        "Web3.Types",
-        "Web3.Utils",
-        "Web3.Decoders",
-        "Web3.Encoders",
-        "Web3.Eth",
-        "Web3.Eth.Contract",
-        "Web3.Eth.Accounts",
-        "Web3.Eth.Wallet",
-        "Web3.Eth.Subscribe"
+        ".",
+        "./src",
+        "./sandbox",
+        "./examples/lightbox",
+        "./examples/testbox",
+        "./test"
     ],
+    "exposed-modules": [],
     "native-modules": true,
     "dependencies": {
-      "elm-lang/core": "5.1.1 <= v < 6.0.0",
-      "gilbertkennen/bigint": "1.0.1 <= v < 2.0.0",
-      "NoRedInk/elm-decode-pipeline": "3.0.0 <= v < 4.0.0"
+        "Chadtech/elm-bool-extra": "1.1.1 <= v < 2.0.0",
+        "NoRedInk/elm-decode-pipeline": "3.0.0 <= v < 4.0.0",
+        "elm-lang/core": "5.1.1 <= v < 6.0.0",
+        "elm-lang/html": "2.0.0 <= v < 3.0.0",
+        "gilbertkennen/bigint": "1.0.1 <= v < 2.0.0",
+        "mdgriffith/style-elements": "4.0.0 <= v <= 5.0.0",
```

Comment:
We can take style-elements out since it's only in the test/example app now.

Reply:
Also, can we just bump package version to 0.3.0 instead of 1.0? I know this doesn't follow standard semantic versioning, since there was definitely a breaking change in `isAddress`. But this is still alpha/beta and don't really want to consider it 1.0 yet.

Reply:
Yup - will remove those deps and bump the package